site/styles/_pages.scss

```diff
@@ -3,6 +3,8 @@
   @include media-breakpoint-up(md){
     padding-bottom: $grid-gutter-width;
   }
+  flex: 1 0 auto;
```

Comment:
Is this doing the same as the removed `margin: 0 auto`?

Reply:
It is shorthand for: flex-grow, flex-shrink, flex-basis. The .page class is only as high as its content. Setting the .page to ‘flex-grow: 1’ makes it grow to take up all of the remaining space in the parent flex container (body). This is what pushes the footer to the bottom of the page.
(https://cssreference.io/property/flex-grow/)

Reply:
I was using the flexbox section of this article here for the fix (https://css-tricks.com/couple-takes-sticky-footer/). From my testing however, it seems like flex-shrink is unnecessary (as well as the auto).

Reply:
Okay, my comment was that `margin: 0 auto` is used to center the content - if it's removed, doing it some other way?

Reply:
It is still centered because there is already another `margin: 0 auto` inside the form container. I have only removed it from the parent. I did this because when I changed `.body` to flexbox, `margin: 0 auto` was squashing the form.

Reply:
Gotcha 👍